src/hooks/invitation.ts

```diff
@@ -20,11 +20,11 @@ export default (queryConfig: QueryClientConfig) => {
         }
         return Api.getInvitation(queryConfig, id);
       },
+      meta: {
+        typeError: getInvitationRoutine.FAILURE,
```

Comment:
```suggestion
        routine: getInvitationRoutine,
```
I think this value is always available, so I think it makes sense to give the full routine, then you can get the failure from it?